test/analysis/test_error_analysis.py

```diff
@@ -2,20 +2,33 @@
 
 import numpy as np
 
-from snorkel.analysis.error_analysis import error_buckets
+from snorkel.analysis.error_analysis import label_buckets
 
 
 class ErrorAnalysisTest(unittest.TestCase):
-    def test_error_buckets(self) -> None:
-        golds = np.array([1, 2, 3, 1, 2, 3])
-        preds = np.array([[2], [1], [3], [1], [1], [3]])
-        buckets = error_buckets(golds, preds)
+    def test_label_buckets(self) -> None:
```

Comment:
Since we now support more than two arguments, there should be at least one test case for that.

Reply:
the third one does that